### PR TITLE
Fix reverse logic issues for Horizontal awnings

### LINKED ISSUE #103: io:HorizontalAwningIOComponent reports 0 as the current_position when it's fully opened
**Describe the bug**
The `current_position` of the `io:HorizontalAwningIOComponent` is always `0` but the `core:DeploymentState` is `100` when:
* Using the service `cover.open_cover`
* Setting the `100` position using the service `cover.set_cover_position`

Using the service `cover.set_cover_position` to establish the `99` position reports:
```yaml
current_position: 100
core:DeploymentState: 98
```

Using the service `cover.set_cover_position` to establish other positions, like `50` reports the correct `current_position` and the same `core:DeploymentState`.

Despite of having the `custom_components.tahoma: debug` configuration, no logs are registered during the reproduction of this bug.

**To Reproduce**
1. Send this request:
```yaml
service: cover.set_cover_position
data:
  entity_id: cover.myawning
  position: 100
```
2. The `current_position` attribute of the `cover.myawning` entity reports the value `0`
3. The slider in the GUI is in the `0` position.
4. The close button in the GUI is not enabled.

**Expected behavior**
The `current_position` attribute of the `cover.myawning` entity must report the value `100`. and the same with the slider in the GUI and the close button must be enabled when the awning is at full opening position.

**Environment**
 - **Home Assistant version**: 0.112.2
 - **Platform**: Somfy TaHoma 1.5 (custom component)

**Device:**
 - **Model**: PositionableHorizontalAwning by Somfy
 - **Type**: io:HorizontalAwningIOComponent
 - **Device details**: sorry, I can't access the `https://www.tahomalink.com/enduser-mobile-web/steer-html5-client/tahoma/` to get the details of the device. I'm using the same credentials that I use in this platform, but the web reports an error (maybe because I own a Connexoon and not a Tahoma).

**Additional context**
The problems with this awning started after a Connexoon firmware update originally reported in the [home-assistant/core issue #37291](https://github.com/home-assistant/core/issues/37291). I haven't managed to retrieve the firmware version in the Connexoon Terrace Android app.

### PR DESCRIPTION
**Pull request by iMicknl**
Fix #103

### CODE REVIEW

File: custom_components/tahoma/cover.py [hunk 1]
```diff
@@ -131,10 +134,6 @@ async def async_set_cover_position(self, **kwargs):
         """Move the cover to a specific position."""
         position = 100 - kwargs.get(ATTR_POSITION, 0)
 
-        # HorizontalAwning devices need a reversed position that can not be obtained via the API
-        if "Horizontal" in self.device.widget:
-            position = kwargs.get(ATTR_POSITION, 0)
-
```
Comment (tetienne) [inline custom_components/tahoma/cover.py:137]: Why remove this part?

Comment (iMicknl) [reply custom_components/tahoma/cover.py:137]: Based on https://github.com/iMicknl/ha-tahoma/issues/103#issuecomment-663761420.

Comment (tetienne) [reply custom_components/tahoma/cover.py:137]: Hum, oki. I should try also with own Awning.

Comment (iMicknl) [reply custom_components/tahoma/cover.py:137]: @tetienne do you have a Horizontal Awning as well? I am not sure about this change, hence I made it a draft and looking for testers.

Comment (tetienne) [reply custom_components/tahoma/cover.py:137]: Yes I have this one: io:HorizontalAwningIOComponent. I can test at noon.

Comment (tetienne) [reply custom_components/tahoma/cover.py:140]: Just test it, and the behavior is not good. The previous one was ok.

Comment (iMicknl) [reply custom_components/tahoma/cover.py:140]: @tetienne thanks for confirming. This brings me back to the drawing table. Weird that some devices have a totally different behavior.

**Pull request closed without merging**